apps/server/src/graphql.module.ts

```diff
@@ -32,6 +32,8 @@ import { Metrics } from './metrics/metrics';
             res,
           }),
           plugins: [new GQLLoggerPlugin(metrics)],
+          // only able to use playground in dev mode
+          playground: config.affine.canary,
```

Comment:
should disable introspection at the same time if you want to disable playground in prod